operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -48,6 +52,7 @@
 public class OperatorMetricsTest {
     private static Vertx vertx;
 
+
```

Comment:
Do we need the double empty line?

Reply:
Here you for some reason add more empty lines again.

Reply:
I guessI have made wrong configuration in my ide --> It sometimes add additional lines. I will remove the unnecessory spaces and make the indentation configs proper there